cdc/entry/schema_storage.go

```diff
@@ -595,6 +632,47 @@ func (s *Storage) CloneTables() map[uint64]TableName {
 	return mp
 }
 
+func (s *Storage) Clone() *Storage {
```

Comment:
What is this `Clone` used for, some fileds still hold the same address of the original `*Storage`, such as `resolvedTs`, `currentJob`

Reply:
https://github.com/pingcap/ticdc/pull/294/files#diff-f4b6dbaac7d78b49e0dc5a4a0d95a2fdR91

the `Clone` is used in `StorageBuilder`, it's expected that `resolvedTs` and `currentJob` hold the same address.